Neos.Flow/Migrations/Code/Version20201109224100.php

```diff
@@ -33,7 +33,7 @@ public function getIdentifier()
     public function up()
     {
         $this->searchAndReplace('Doctrine\DBAL\Migrations\AbstractMigration', 'Doctrine\Migrations\AbstractMigration', ['php']);
-        $this->searchAndReplaceRegex('/public function getDescription\(\)(\s*\{)?$/m', 'public function getDescription(): string $1', ['php']);
+        $this->searchAndReplaceRegex('/(Neos\\\Flow\\\Persistence\\\Doctrine\\\Migrations.+)public function getDescription\(\)(\s*\{)?$/sm', '$1public function getDescription(): string $2', ['php']);
```

Comment:
Thanks for this! Two questions: Aren't there migrations that do not live in this namespace? Also this will capture cases where the Migrations namespace is imported, but the actual class itself is not a Migration.

Still an improvement over the current loose migration regex

Reply:
> Aren't there migrations that do not live in this namespace

In our Doctrine Service class `Neos\Flow\Persistence\Doctrine\Migrations` is defined as the migraiton namespace. I consider this to be a safe approach

> Also this will capture cases where the Migrations namespace

That is a good point. Add a `namespace ` to the start of the line?

Reply:
Thanks for the comments! I added `namespace ` to the regex.